scripts/validate_docstrings.py

```diff
@@ -345,9 +343,7 @@ def header(title, width=80, char="#"):
         full_line = char * width
         side_len = (width - len(title) - 2) // 2
         adj = "" if len(title) % 2 == 0 else " "
-        title_line = "{side} {title}{adj} {side}".format(
-            side=char * side_len, title=title, adj=adj
-        )
+        title_line = f"{char * side_len} {title}{adj} {char * side_len}"
```

Comment:
can you do the multiplication on the previous line

Reply:
I think this is OK for now (we have a pretty large queue) but if a sticking point can certainly be done as a follow up